src/test/run-make/thumb-none-qemu/example/src/main.rs

```diff
@@ -1,19 +1,17 @@
 // #![feature(stdsimd)]
 #![no_main]
 #![no_std]
-
-extern crate cortex_m;
-
-extern crate cortex_m_rt as rt;
-extern crate cortex_m_semihosting as semihosting;
-extern crate panic_halt;
-
+#![deny(rust_2018_idioms)]
 use core::fmt::Write;
 use cortex_m::asm;
-use rt::entry;
+use cortex_m_rt::entry;
+use cortex_m_semihosting as semihosting;
 
-entry!(main);
+//FIXME: This imports the provided #[panic_handler].
+#[allow(rust_2018_idioms)]
+extern crate panic_halt;
```

Comment:
Is this a known issue?

Reply:
I suspect so -- probably a bug, but we can leave it like this for now.

Reply:
OK.